content/departments/product-engineering/engineering/process/incidents/playbooks/ci.md

```diff
@@ -217,22 +204,6 @@ In order to handle problems with the CI, the following elements are necessary:
      1. **Yes**, that's a flake. See the [Spotted a flake scenario](#spotted-a-flake)
    - **No**: it's not a flake, reach out the team owning those tests.
 
-### Restarting the agents
```

Comment:
Doesn't have to be this PR, but you might still want to restart agents (e.g. if a dispatch is borked) by deleting an entire dispatched Kubernetes job

Reply:
yeah, so in typical fashion I was going to leave some comments here asking for feedback...but pushed and then went off on something else. 

So, my gut feel is we want to probably want to reference deleting a job entirely if a series of agents that have been deployed is failing. 

I'll follow up here.